sfge/src/main/java/com/salesforce/config/UserFacingErrorMessages.java

```diff
@@ -0,0 +1,14 @@
+package com.salesforce.config;
+
+/**
+ * Contains error message constants that will be displayed to users. TODO: move all other
+ * user-facing messages here.
+ */
+public final class UserFacingErrorMessages {
```

Comment:
Just a simple constants file at the moment, but as we move more messages to here, would be easier to switch to a different mechanism.